README.md

```diff
@@ -1,18 +1,52 @@
-## run-dljc
+## wpi-many
 
-This project contains scripts used to control [do-like-javac](https://github.com/kelloggm/do-like-javac) to run large-scale experiments using Checker Framework whole-program inference.
+This project contains scripts used to control
+[do-like-javac](https://github.com/kelloggm/do-like-javac) to run
+[whole-program
+inference](https://checkerframework.org/manual/#whole-program-inference)
+(WPI) of Checker Framework annotations.
 
-The most important script is `run-dljc.sh`. It runs whole-program inference on every buildable gradle or maven project in a list of
-(GitHub repository URL, git hash) pairs.  The output is stored in a results directory, which can be summarized
-with `summary.sh`. To create an input file, you can use `run-queries.sh` to search GitHub for candidate repositories, or you can write
-an input file from scratch.
+There are two modes these scripts support:
+1. running large-scale experiments on a collection of GitHub repositories, and
+2. running WPI on a single project on your local machine.
 
-Three example files are included:
-* `no-literal-securerandom-exact-dljc-cmd.sh` is a no-arguments script that serves as an example of how to use `run-dljc.sh` with a
-custom checker.
-* `securerandom.list` is a list of repositories and hashes in the format expected by `run-dljc.sh`. It was created by invoking `run-queries.sh`
-and is used by `no-literal-securerandom-exact-dljc-cmd.sh`.
-* `securerandom.query` is the GitHub search query used to create `securerandom.list`. The exact command to create `securerandom.list` is
-`./run-queries.sh securerandom.query 100`.
+### Large experiments
+
+To run a large experiment:
+
+1. Use `query-github.sh` to search GitHub for candidate repositories.
+File `securerandom.query` is an example query, and file `securerandom.list`
+was created by running `./query-github.sh securerandom.query 100`.
+
+2. Use `wpi-many.sh` to run whole-program inference on every buildable
+gradle or maven project in a list of (GitHub repository URL, git hash)
+pairs.
+ * If you are using a checker that is distributed with the Checker
+   Framework, use wpi-many.sh or wpi.sh directly.
+ * If you are using a checker that is not distributed with the Checker
+   Framework (also known as as "custom checker"), file
+   `no-literal-securerandom-exact-cmd.sh` is a no-arguments
+   script that serves as an example of how to use `wpi-many.sh`.
+
+3. Use `summary.sh` to summarize the logs in the output results directory.
+Use its output to guide your analysis of the results of running ./wpi-many.sh:
+you should manually examine the results of any project that appears in the
+"unaccounted for" list it produces.
```

Comment:
What does "unaccounted for" mean?  Can you define it and/or use a more descriptive term?